internal/controller/kyma_controller.go

```diff
@@ -539,6 +541,9 @@ func (r *KymaReconciler) deleteManifest(ctx context.Context, trackedManifest *v1
 
 func (r *KymaReconciler) UpdateMetrics(ctx context.Context, kyma *v1beta2.Kyma) {
```

Comment:
Can you verify if the current updateMetrics is only for `is-kyma-managed` mode? This error is only relevant for kyma in KCP, for other setup, there is no need to send warning event.